.github/scripts/build-images.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# @authors: Somya Arora, Arash Pourhabibi
+# @modified: Shanqing Lin
+
+
+# 1. Figure out the modified files
+if [ "${GITHUB_EVENT_NAME}" = "pull_request" ]; then
+  modified_files=$(git --no-pager diff --name-only ${PR_COMMIT_RANGE})
+else
+  modified_files=$(git --no-pager diff --name-only ${PUSH_COMMIT_RANGE})
+fi
+# 2.Get benchmark name and tag name for this build
+benchmark_name=${DH_REPO#*/}
+tag_name=$IMG_TAG
+if [ -z "$modified_files" ]; then
+    echo "No Modifications required."
+else
+    echo "Checking against modified files"
+fi
+# 3.Find out whether the files related with the current build were modified or not
+if (grep -q "$benchmark_name/$tag_name" <<<$modified_files) ||
+    (grep -q "$benchmark_name" <<<$modified_files && [ $tag_name = "latest" ]) ||
+    (grep -q "build-images.sh" <<<$modified_files) ||
+    (grep -q "build-images.yaml" <<<$modified_files) || 
+    ( [ $benchmark_name = "debian" ] && (grep -q "commons/base-os" <<<$modified_files) ); then
+    # if modified, then rebuild their docker image
+    docker buildx prune -a -f
+    # reference: https://github.com/docker/buildx/issues/495
+    docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+    docker buildx create --name multiarch --driver docker-container --use
+    docker buildx inspect --bootstrap
+
+    if [ $benchmark_name = "debian" ]; then
+        cd commons/base-os
+        for arch in amd64 arm64; do
```

Comment:
Why is `RISC-V` missing? we had this before.

Reply:
It would trigger a building error when trying to changing the root. I think we agree to ignore it before completing the Github action, right

Reply:
In your PR's comment you have mentioned "add support for ARM and RISC-V", so I assume this has to be fixed for this PR.